### DISCUSSION TITLE
Highlight Focus code line style by default or plugin

### DISCUSSION THREAD

**Discussion by AbraaoAlves** [marp-team/marp#511] [Ideas]
It would be amazing if there were a way to display code using syntax like this

![image](https://github.com/marp-team/marp/assets/608731/6ff7110a-e3e0-40c2-9d29-1d29bec71190)

``` md 
``` js@focus=3:6,14
```

It'll be possible if we can simply add HighlightJS plugin like it: https://github.com/huytd/highlightjs-focus

**Comment by rdhar** [comment]
This would be great to have, and one of the reasons that reveal.js is still used due to its [highlight-lines bracket syntax](https://revealjs.com/markdown/#syntax-highlighting).
